docs/index.rst

```diff
@@ -52,11 +58,20 @@ Supported Devices
 * D-Link
 
   - DIR-825 (B1)
+  - DIR-615 (C1)
+
+* GL-Inet
```

Comment:
Please test your changes before opening pull requests...

You need to add an empty line before starting a new sublist. This mistake was made all over the file...